dotcom-rendering/src/server/lib/get-content-from-url.js

```diff
@@ -14,9 +14,20 @@ async function getContentFromURL(_url) {
 		// Reconstruct the parsed url adding .json?dcr which we need to force dcr to return json
 		const jsonUrl = `${url.origin}${url.pathname}.json?dcr=true&${searchparams}`;
 
-		const { html, ...config } = await fetch(jsonUrl).then((article) =>
-			article.json(),
-		);
+		// Explicitly pass through GU headers - this enables us to override properties such as region in CI
+		let guHeaders = {};
+		if (_headers) {
+			guHeaders = Object.keys(_headers)
+				.filter((key) => key.toLowerCase().startsWith('x-gu-'))
+				.reduce((result, key) => {
+					result[key] = _headers[key];
+					return result;
+				}, {});
+		}
+
+		const { html, ...config } = await fetch(jsonUrl, {
```

Comment:
At this point in time, I only saw the need to forward the headers. Maybe we'd also want to forward the cookies too? For now, this might be enough.